pages/about.jsx

```diff
@@ -7,79 +7,107 @@ var Illustration = require('../components/illustration.jsx');
 
 var config = require('../config/config');
 
-var Intro = (
-  <div>
-    <h1>About the Mozilla Learning Network</h1>
-    <section className="intro">
-      <Illustration width={226} height={226}
-      src1x="/img/pages/about/hive-nyc-summer-quest.jpg"
-      alt="HiveLearningNYC 2014 Hive NYC Summer Quest Maker Party">
-        <p>
-          We are cultivating a global network of web literacy leaders who are teaching and
-          learning the most important skills of our age: the ability to read, write and
-          participate in the digital world. Our programs and resources can help you make
-          an impact in your community, and participate in an openly networked world.
-        </p>
-      </Illustration>
-    </section>
-  </div>
-);
+var Intro = React.createClass({
+  contextTypes: {
+    intl: React.PropTypes.object
+  },
+  render() {
+    return (
+      <div>
+        <h1>About the Mozilla Learning Network</h1>
+        <section className="intro">
+          <Illustration width={226} height={226}
+          src1x="/img/pages/about/hive-nyc-summer-quest.jpg"
+          alt="HiveLearningNYC 2014 Hive NYC Summer Quest Maker Party">
+            <p>
+              We are cultivating a global network of web literacy leaders who are teaching and
+              learning the most important skills of our age: the ability to read, write and
+              participate in the digital world. Our programs and resources can help you make
+              an impact in your community, and participate in an openly networked world.
+            </p>
+          </Illustration>
+        </section>
+      </div>
+    )
+  }
+});
 
-var OurFocus = (
-  <section className="row list-with-illust">
-    <Illustration width={199} height={199}
-    src1x="/img/pages/home/svg/icon-circle-home.svg"
-    alt="A graphic showing an illustration of a person, a book, some gears, a map pin, and a square acedemic cap">
-      <ul className="colored-list">
-        <li>Learn and deepen your 21st Century digital skills, and get better at teaching them to others.</li>
-        <li>Contribute to real-world products and projects that are open, participatory and networked.</li>
-        <li>
-          Access free resources like event planning guides and step-by-step teaching activities, or share
-          your own resources to receive feedback from peers.
-        </li>
-        <li>Develop your skills in facilitative leadership, participatory practices and collaboration.</li>
-      </ul>
-    </Illustration>
-  </section>
-);
+var OurFocus = React.createClass({
+  contextTypes: {
+    intl: React.PropTypes.object
```

Comment:
Not at the moment, but I was testing to see if this will work after my integration.